src/test/java/com/imsweb/x12/reader/X12ReaderTest.java

```diff
@@ -46,12 +48,52 @@ public void testConstructors() throws IOException {
         assertEquals(fromFileUtf8.getLoops().get(0).toString(), fromReaderUtf8.getLoops().get(0).toString());
     }
 
+    /**
+     * Here we will test that you can go from x12, make changes, then serialize the
+     * x12 once again.
+     */
+    @Test
+    public void testSerializeBasic() throws IOException {
+        URL url = this.getClass().getResource("/837_5010/x12_valid.txt");
+
+        X12Reader fromFileUtf8 = new X12Reader(FileType.ANSI837_5010_X222, new File(url.getFile()),
+                StandardCharsets.UTF_8);
+
+        String expected = IOUtils
+                .toString(this.getClass().getResourceAsStream("/837_5010/x12_valid.txt"), StandardCharsets.UTF_8)
+                .trim();
+        Assert.assertEquals(expected, fromFileUtf8.toX12String(LineBreak.LF).trim());
```

Comment:
Overall I think things look good. Can we add the same check about the line breaks in this unit here like you did below? This test failed when I was looking at things locally but I think adding the check should fix it.